### PR TITLE
Contributing update

### PR DESCRIPTION
**Pull request by rowanwins**
Update the contributing.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,20 +1,36 @@
-## Generating new documentation
-_Important: to get the submodule of Turf going, you'll need to clone this repository with
-the `--recursive` flag._
+## Updating existing documentation for a module
+- This happens in the [Turf repository](https://github.com/turfjs/turf/) with `/pacakges/` directory for each module.
```
Comment (twelch) [inline CONTRIBUTING.md:2]:
```suggestion
- This happens in the [Turf repository](https://github.com/turfjs/turf/) with `/packages/` directory for each module. 
```

**Review by twelch — APPROVED** [review]
I haven't walked through the steps to verify but they LGTM.  Glad to see this improvement.

**Comment by rowanwins** [comment]
If there is anything I've learned from being involved in Turf it that it's good to have someone else who's walked through the process of doing something. Perhaps we leave this sitting here until we're ready to do a minor docs update and then @twelch can have a go at actually walking through it to check it is sufficient.

**Comment by twelch** [comment]
Sure.  I'm okay with follow-on PRs too for corrections.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,20 +1,36 @@
-## Generating new documentation
-_Important: to get the submodule of Turf going, you'll need to clone this repository with
-the `--recursive` flag._
+## Updating existing documentation for a module
+- This happens in the [Turf repository](https://github.com/turfjs/turf/) with `/packages/` directory for each module. 
+- To fix the documentation for a module you need to modify the [jsdoc](https://jsdoc.app/) for an individual module found in either the `index.js` or the `index.ts`. 
+  - eg to update the documentation for `@turf/along` you'd need to modify the jsdoc within `/turf/packages/turf-along/index.ts`.
+- Once you've made your changes you'll need to create a pull request to the [Turf repository](https://github.com/turfjs/turf/).
+  - **Note** Nothing needs to be changed in the `README.md` for a module.
+- Even once your PR is accepted the docs will still need to be updated manually.
```
Comment (mfedderly) [inline CONTRIBUTING.md:7]: `Even once` to just `Once`

**Review by mfedderly — APPROVED** [review]
LGTM

**Comment by JamesLMilner** [comment]
Can we merge this? @rowanwins

**Pull request merged**